datascience/tables.py

```diff
@@ -147,9 +147,11 @@ def _add_column_and_format(self, table, label, column):
             table._formats[label] = self._formats[label]
 
     @classmethod
-    def from_df(cls, df):
+    def from_df(cls, df, keep_index=False):
```

Comment:
Please add a docstring to explain what `keep_index` does (it creates a column named `index`, etc.)

Reply:
Done!